src/i18n.js

```diff
@@ -0,0 +1,34 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+
+import translationDE from './translations/de/translation.json';
```

Comment:
imo, you could simplify the path to translations/de.json, translations/en.json, etc.
name the variable in plural form: translationsDe/translationsDE